rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1246,9 +1246,13 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	if (!game)
 	{
 		SubscribeTooltip(ui->gs_resizeOnBoot, tooltips.settings.resize_on_boot);
+		SubscribeTooltip(ui->gb_gs_height, tooltips.settings.resize_on_boot);
+		SubscribeTooltip(ui->gb_gs_width, tooltips.settings.resize_on_boot);
 		SubscribeTooltip(ui->gs_disableMouse, tooltips.settings.disable_mouse);
 		SubscribeTooltip(ui->gs_disableKbHotkeys, tooltips.settings.disable_kb_hotkeys);
 		SubscribeTooltip(ui->gs_showMouseInFullscreen, tooltips.settings.show_mouse_in_fullscreen);
+		SubscribeTooltip(ui->gs_hideMouseOnIdle, tooltips.settings.hide_mouse_on_idle);
```

Comment:
It would be better to use the tooltip on the encompassing widget, because like this you may see the tooltip going back to default while you hover in the wrong spot between the items

Reply:
I'm absolutely with you on that... and it's probably me just not knowing what tooltips can actually be assigned to.
This turns out to be where the mouse idle hide tooltip went... I was trying to be too smart, and had assigned it to the layout box in the .UI file code (yeah, I skimmed right on past the 'This file will be overwritten...' part at the top).  But when I next saved it in Qt, Qt says 'NO!' layout boxes cannot be assigned tooltips.

Hence this became the 'next best thing'.  Probably a proper group box or something would be the way, but I really did try that and it just made the layout engine go mental.  Which I suspect was more me not doing something right...

Reply:
You can simply wrap the layout in a QWidget and put the tooltip on that

Reply:
"simply"... hmm
The default Qwidget wants to inherit its padding settings, and push them down to its contained children, which messes up the layout of the checkboxes and sub items etc unless the padding is overridden on the QWidget..
Which ain't so simple for Qt neophyte like myself.

If you want to PR on 'the proper way', would be happy to see it.

Reply:
Sure. Later